rts/System/Sound/OpenAL/Sound.cpp

```diff
@@ -468,8 +476,12 @@ void CSound::OpenLoopbackDevice(const std::string& deviceName)
 	if (alcLoopbackOpenDeviceSOFT == nullptr || alcIsRenderFormatSupportedSOFT == nullptr || alcRenderSamplesSOFT == nullptr)
 		return;
 
-	if (!configHandler->GetBool("UseSDLAudio"))
+	if (!configHandler->GetBool("UseSDLAudio")) {
+		LOG("[Sound::%s] UseSDLAudio is NOT set, falling back to openal-soft backends", __func__);
 		return;
+	} else {
+		LOG("[Sound::%s] UseSDLAudio is set", __func__);
```

Comment:
Ditto for above, we're more interested in logging the cases where compatibility is poorer, in this case using sdl audio logging might not be necessary.